explorer/frontend/api/transactions.js

```diff
@@ -324,21 +324,46 @@ const formatAccountKeyLink = (data, filter) => {
 const formatMultisigTransaction = (data, filter) => {
 	const rawEmbeddedTransaction = {
 		...data,
-		transactionType: data.embeddedTransactions[0].transactionType
+		transactionType: data.embeddedTransactions[0].transactionType,
+		fee: data.embeddedTransactions[0].fee
 	};
 	if (rawEmbeddedTransaction.transactionType === TRANSACTION_TYPE.TRANSFER)
 		rawEmbeddedTransaction.value = [{ message: data.embeddedTransactions[0].message }, data.embeddedTransactions[0].mosaics];
 	else
 		rawEmbeddedTransaction.value = data.embeddedTransactions;
 
+	const formattedTransaction = formatBaseTransaction(data, filter);
 	const formattedEmbeddedTransaction = transactionFromDTO(rawEmbeddedTransaction, filter.address);
 
+	// Fees breakdown
+	const multisigFee = truncateDecimals(formattedTransaction.fee, config.NATIVE_MOSAIC_DIVISIBILITY);
+	const embeddedTransactionsFee = truncateDecimals(formattedEmbeddedTransaction.fee, config.NATIVE_MOSAIC_DIVISIBILITY);
+	const signaturesFee = truncateDecimals(
+		data.embeddedTransactions[0].signatures.reduce((total, signature) => total + (signature.fee || 0), 0),
```

Comment:
Is there a case where a signature does not have a fee? 🤔  ``(signature.fee || 0)``